package.json

```diff
@@ -2,9 +2,9 @@
     "name": "jwt-decode",
     "version": "3.1.2",
     "description": "Decode JWT tokens, mostly useful for browser applications.",
-    "main": "build/jwt-decode.cjs.js",
-    "module": "build/jwt-decode.esm.js",
-    "types": "index.d.ts",
+    "main": "build/cjs/jwt-decode.js",
+    "module": "build/esm/jwt-decode.js",
+    "types": "build/typings/index.d.ts",
```

Comment:
Might be worth it to use the [`exports` field](https://nodejs.org/api/packages.html#exports) here as well. It will encapsulate internal unexported modules, so that they cannot be imported by accident (see also the [WebPack docs](https://webpack.js.org/guides/package-exports/) on the subject).